src/app.js

```diff
@@ -323,6 +323,9 @@ app.routeAsync("/groups/:groupName")
 app.use("/groups/:groupName/settings",
   endpoints.groups.setGroup(req => req.params.groupName));
 
+app.routeAsync("/groups/:groupName/settings")
+  .getAsync(endpoints.static.sendGatsbyEntrypoint);
```

Comment:
I was originally wary of `sendGatsbyPage()` because of the comment:

> sendGatsbyPage("groups/:groupName/index.html") should work, but it
    renders wrong for some reason that's not clear.

However, I just tried it and I'm not seeing anything weird. I'll go with `sendGatsbyPage()` and update the groups page route as well.

Reply:
Updated in 635c9e7d5b2cdc55194160472f6d0e619874391a

Reply:
Wait, I take it back...the individual groups pages are rendering weird for me at [dev.nextstrain.org](https://dev.nextstrain.org/groups/blab)
![Screen Shot 2023-04-14 at 5 58 16 PM](https://user-images.githubusercontent.com/40774762/232175247-8c408c59-fb2e-4150-9cd4-890e49da4101.png)

Oh, and the `Return to ${groupName} Page` button points to `/groups/undefined`. 

I'll try digging a little into this on Monday.

Reply:
I am able to replicate this locally when I run `npm run server`. I didn't see it before because I was testing with `npm run dev`. 

What's happening is the Groups page is being rendered within the `FixedBanner` div, which I think comes from the `ErrorBanner` that would display if the app could not find the group. I not sure _why_ though...

Reply:
I'm leaving this as is with `sendGatsbyEntrypoint` for now because I have no idea what Gatsby is doing behind the scenes here to end up rendering the page within the `FixedBanner` div.